drivers/mfd/mfd_axp192.c

```diff
@@ -50,12 +153,495 @@ static int mfd_axp192_init(const struct device *dev)
 	return 0;
 }
 
+int mfd_axp192_gpio_func_get(const struct device *dev, uint8_t gpio, enum axp192_gpio_func *func)
```

Comment:
question: why is this part of the mfd layer? who else is using this?

Reply:
It is used by regulator and gpio drivers as some pins of the pmu are multiplexed. And it also was a recommendation in my regulator PR (https://github.com/zephyrproject-rtos/zephyr/pull/58809#discussion_r1217764673) .